src/components/data-management/AnalysisModal.jsx

```diff
@@ -48,15 +44,13 @@ const AnalysisModal = (props) => {
     setExperimentsList(updatedList);
   }, [activeProject, experiments]);
 
-  const onLaunchAnalysis = (experimentId) => {
+  const onLaunchAnalysis = async () => {
     onLaunch(experimentId);
-    const analysisPath = '/experiments/[experimentId]/data-processing';
     const lastViewed = moment().toISOString();
     dispatch(updateExperiment(experimentId, { lastViewed }));
     dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
-
-    dispatch(runGem2s(experimentId));
-    router.push(analysisPath.replace('[experimentId]', experimentId));
+    const analysisPath = '/experiments/[experimentId]/data-processing';
+    window.location.href = await analysisPath.replace('[experimentId]', experimentId);
```

Comment:
Why use `window.location.href` and make it async instead of using the router?

Reply:
these are changes introduced by Marcell from this PR -https://github.com/biomage-ltd/ui/pull/499

Reply:
again, do figure that out